AdapterInterface.php

```diff
@@ -75,7 +75,7 @@ public function compareAutoRouteContent(AutoRouteInterface $autoRoute, $contentO
      *
      * @return null|Symfony\Cmf\Component\Routing\RouteObjectInterface
      */
-    public function findRouteForUri($uri);
+    public function findRouteForUri($uri, UriContext $uriContext);
```

Comment:
`$uriContext` has a `$uriContext->getUri()` method which will return the `$uri` passed above. I guess it makes more sense to simply replace it with the `$uriContext`.